website/content/docs/connect/config-entries/service-defaults.mdx

```diff
@@ -421,6 +421,16 @@ represents a location outside the Consul cluster. They can be dialed directly wh
               description:
                 'The namespace of the upstream. This should not be set to the wildcard specifier `*`.',
             },
+            {
+              name: 'Peer',
+              type: 'string: ""',
+              description: 
+                `The name of the peer containing the upstream. This should not be set to the wildcard specifier \`*\`.<br><br>
+                    NOTE: If the \`peer\` field is not set in any of \`Overrides\`, an override applies to peered services with the same [\`name\`](/consul/docs/connect/config-entries/service-defaults#name) 
+                    regardless if this field is set. This maintains backward compatibility with Consul 1.14.0. 
+                    If any override has the \`peer\` field specified, this compatibility mode is disabled and each peered service must have it's own override.
+                    This compatibility mode will be removed in a future version of Consul.`,
```

Comment:
```suggestion
                `The name of the peer containing the upstream. Do not use a wildcard specifier ( \`*\`).<br><br>
                    If the \`peer\` field is not set in an \`Overrides\` configuration, then Consul applies overrides to peered services with the same [\`name\`](/consul/docs/connect/config-entries/service-defaults#name),
                    even if the \`peer\` field is set in the service configuration. This maintains backward compatibility with Consul 1.14.0. 
                    If the \`peer\` field is specified in any override, then backward compatibility with Consul 1.14.0 is disabled. As a result, each peered service must have an override configuration.` ,
```
A few suggestions to switch to active voice and to remove temporal language per the style guide. Also, I was a little confused by the "even if this field is set" part of the NOTE statement. It reads like contradictory statements. I tried to add something that made sense, but I don't think it's correct. Do we mean that the peer field must be set for all overrides, otherwise the override configuration is applied to any service name matching the configuration in any peered cluster?

Reply:
Actually I think this is pretty close to what I was trying to say, I think we just need to remove `even if the \`peer\` field is set in the service configuration` for clarity.